src/main/java/com/bookstore/entity/Book.java

```diff
@@ -39,6 +39,6 @@ public class Book {
 
     private String coverImage;
 
-    @Column(nullable = false)
-    private boolean isDeleted;
+    @Column(columnDefinition = "TINYINT(4)", name = "is_deleted")
```

Comment:
What is the reason to make it TINYINT?

Reply:
An error occurs if I get rid of the "columnDefinition" line, I am not sure why but I think it is something related to how hibernate maps fields to the DB columns, probably boolean is represented with a tinyint in mysql, but I changed it to the length of 1 in the new commit

Reply:
Schema-validation: wrong column type encountered in column [is_deleted] in table [books]; found [tinyint (Types#TINYINT)], but expecting [bit (Types#BOOLEAN)]